packit_service/models.py

```diff
@@ -1119,6 +1120,49 @@ def get_range(cls, first: int, last: int) -> Optional[Iterable["TFTTestRunModel"
             ]
 
 
+class ProjectTokenModel(Base):
```

Comment:
The name does not match the current content. We don't save tokens any more.

Reply:
How about `ProjectAuthenticationIssueModel`

Reply:
Sounds good.

Reply:
Don't forget to rename table as well...